app/src/main/java/com/spoony/spoony/data/dto/request/RegisterPostRequestDto.kt

```diff
@@ -0,0 +1,19 @@
+package com.spoony.spoony.data.dto.request
+
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+// 문제 없음.
+@Serializable
+data class RegisterPostRequestDto(
+    @SerialName("userId") val userId: Long,
+    @SerialName("title") val title: String,
```

Comment:
```suggestion
    @SerialName("userId") 
    val userId: Long,
    @SerialName("title") 
    val title: String,
```
P5: 이런식으로 작성하는건 어떨까요?!

Reply:
반영했습니다~